Aptoide/aptoide-client-v8
# [MOB-124] Stores search suggestions

**What does this PR do?**

This PR aims at fixing the search suggestions in dark theme

**Database changed?**

   No

**Where should the reviewer start?**

- [x] simple_dropdown_item.xml

**How should this be manually tested?**

Open all search views in the app (search, stores search) and confirm that the suggestions are according to the darktheme. Also confirm light theme is correct.

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-124](https://aptoide.atlassian.net/browse/MOB-124)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
